### PR TITLE
Added types of Federated Learning (#205)

### PR DESCRIPTION
**Pull request by burpcat**
## Description
Added the detailed explaination for types of federated learning

## Relevant Issue
Issue #205 

## Affected Dependencies
No dependencies affected

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/NimbleEdge/EnvisEdge/labels)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by burpcat** [comment]
@ramesht007 , Are there any errors in this PR?

File: docs/source/tutorials/Tutorial-Part-1-introduction_to_fl.rst [hunk 1]
```diff
@@ -10,15 +10,59 @@ lifecycle of federated learning and set the standard definitions.
...
+
+
+  Model Centric Federated Learning is further classifid into:
+  
+  * **Cross-Device Federated Learning**
+    
+    When Federated Learning takes place through suitable federated techniques
+    from data across a wide range of devices.
+
+    Typically, Cross-Device FL uses Horizontal Federated Learning
```
Comment (ramesht007) [inline docs/source/tutorials/Tutorial-Part-1-introduction_to_fl.rst:40]: Also Add:

Horizontal Federated Learning is also called `Homogenous Federated Learning`
and also can you explain what is Horizontal Federated Learning here, than below

**Review by ramesht007 — CHANGES_REQUESTED** [review]

**Comment by burpcat** [comment]
Hey @ramesht007 , I made the required changes. could you review again

**Review by ramesht007 — APPROVED** [review]
LGTM!!

**Pull request merged**